src/env.ts

```diff
@@ -11,7 +11,11 @@ export function env (...presets: Preset[]): Environment {
   for (const preset of presets) {
     // Alias
     if (preset.alias) {
-      for (const from in preset.alias) {
+      // Sort aliases from specific to general (ie. fs/promises before fs)
+      const aliases = Object.keys(preset.alias).sort((a, b) =>
+        b.split('/').length - a.split('/').length
```

Comment:
Right now we don't have prefix aliases but same as nitro, we could sort by length as fallback:

```suggestion
        (b.split('/').length - a.split('/').length) || (b.length-a.length)
```